styletesla.css

```diff
@@ -1,79 +1,286 @@
 .body {
-    max-width: 1440px;
+	max-width: 1440px;
     margin: 0 auto;
     }
 .mainpage {
 	height: 100vh;
-        }
+    }
 .header {
 	height: 100px;
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
-}
+	margin-bottom: 40px;
+    }
+ .logo {
+    padding-left: 47px;
+    }
 .navigation a {
-	display: inline;
-    margin-bottom: 67px;
+	font: 15px/18px montserratsemibold;
     cursor: pointer;
     margin-right: 44px;
-}
+    }
 .navigation__link_active {
-	border-bottom: 1px solid red;
-	margin-right: 63px;
-	color: red;}
-
+	padding-bottom: 5px;
+	border-bottom: 1px solid #E51937;
+	color: #E51937;
+    }
 .menu a {
+	cursor: pointer;
+	font: 15px/18px montserratsemibold;
 	margin-right: 30px;
 }
 .main {
 	background: url('img/tesla-wordmark.png') top no-repeat;
+	background-size: contain;
 	display: flex;
 	align-items: center;
 	flex-direction: column;
-}
-
+    }
+.main__title {
+    font: 90px/110px montserratsemibold;
+    margin: 0;
+    }
+.main__subtitle {
+    font: 15px/18px montserratmedium;
+    margin: 0;
+    }	
 .tesla{ 
-	/*width: 67%;*/
 	max-width: 968px;
-
-}
+	margin-bottom: 18px;
+	}
 .description {
-	/*padding: 0;
-	height: 120px;*/
-	width: 600px;
 	display: flex;
 	flex-direction: row;
-	justify-content: space-between;
-}
-
-
-.speed, .road, .wheel {
+	}
+.speed, .road  {
 	display: flex;
- /* переписать классы*/
-}
+	margin-right: 80px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5505628)